Packs/Stamus/Integrations/Stamus/Stamus.yml

```diff
@@ -0,0 +1,109 @@
+category: Network Security
+commonfields:
+  id: Stamus
+  version: -1
+configuration:
+- defaultvalue: https://example.com/
+  display: Stamus Central Server
+  name: url
+  required: true
+  type: 0
+- displaypassword: API Key
+  additionalinfo: The API Key to use for connection
+  name: credentials
+  required: true
+  hiddenusername: true
+  type: 9
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- defaultvalue: '200'
+  display: Maximum number of incidents per fetch
+  name: max_fetch
+  required: false
+  type: 0
+  section: Collect
+- defaultvalue: 3 days
+  display: First fetch time
+  name: first_fetch
+  required: false
+  type: 0
+  section: Collect
+description: '[Get Declaration of Compromises from Stamus Security Platform and build Incidents. Then get related artifacts, events and Host Insight information]'
+display: 'Stamus'
+name: Stamus
+script:
+  commands:
+  - arguments:
+    - name: key
+      default: false
+      description: '[Indicator of Compromise key]'
+      isArray: false
+      required: true
+      secret: false
+    - name: value
+      default: false
+      description: '[Indicator of Compromise value]'
+      isArray: false
+      required: true
+      secret: false
+    deprecated: false
+    description: '[Get events with IOC key/value filter]'
+    execution: false
+    name: Stamus-Check-IOC
+    outputs:
+    - contextPath: StamusIntegration.Output
+      description: '[Fetch events matching an IOC.]'
+      type: String
+  - arguments:
+    - default: false
+      description: '[IP to get Host Insights information]'
+      isArray: false
+      name: ip
+      required: true
+      secret: false
+    deprecated: false
+    description: '[Get Host Insights information]'
+    execution: false
+    name: Stamus-Get-Host-Insight
+    outputs:
+    - contextPath: StamusIntegration.Output
```

Comment:
as mentioned it not a good convention that all the commands would have the same context, each one should have different context output.

if you need any help for the names of the context outputs for each command, let me know